doc/nrfxlib/nrfxlib.doxyfile.in

```diff
@@ -1996,7 +1996,8 @@ PREDEFINED             = "MBEDTLS_CONFIG_FILE=\"@OUTPUT_DIRECTORY@/mbedtls_doxyg
 			 "MBEDTLS_AES_C=" \
 			 "MBEDTLS_CCM_C=" \
 			 "MBEDTLS_GCM_C=" \
-			 "MBEDTLS_CONFIG_FILE="
+			 "MBEDTLS_CONFIG_FILE=" \
+             "OSALCALL="
```

Comment:
What is this ?

Reply:
No clue, looks ok on my machine:
![image](https://user-images.githubusercontent.com/41291385/193227691-6d8f122b-c1de-425c-9d6d-2ab0d0f59ef9.png)

Reply:
Oh, @koffes , you might be asking about the reason of adding this to the doxyfile. This is needed to enable building the LC3 API docs, which all require this variable predefined in the code.

Reply:
As for the formatting, I see you've used spaces instead of tabs to align. In case of misconfigured editor (4 spaces/tabs) it may look ok locally.